src/index.ts

```diff
@@ -1,18 +1,21 @@
 import { Lottery } from "./lottery-application";
-import { writeFile } from 'fs';
-
-console.log("* Initializing Lottery Application *");
-
-var log = '***** Lottery application log *****\n\n';
+import * as fs from "fs";
+import {writeFile} from "fs";
 
 const upperBoundary = 60;
 const numbersToDraw = 6;
+const timestamp = new Date().toISOString();
+const dir = './results';
+const lottery = new Lottery(upperBoundary, numbersToDraw);
+const fileName = `${dir}/lottery-results:${timestamp}.txt`;
```

Comment:
A colon ( : ) is not a good special character for a filename, we can have problems with some OS. Let's adopt a better standard and replace this for a _

Reply:
I will correct that. Yes the naming convention shall be implemented such as:
dir/filename_timestamp.txt
I didnt know that a colon could be problematic for some OS. Thanks!

Reply:
Ok, great. Waiting for new commits. This should work:
`const fileName = `${dir}/lottery-results_${timestamp}.txt`;`

Reply:
Ok, great. Waiting for the new commit.